src/acat/azext_acat/_help.py

```diff
@@ -9,3 +9,38 @@
 # pylint: disable=too-many-lines
 
 from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['acat trigger-evaluation'] = """
+type: command
+short-summary: Triggers quick evaluation for acat report.
+examples:
+  - name: Simple example
+    text: |
+      az acat trigger-evaluation --resource-ids $resources.id
+"""
+helps['acat report download'] = """
+type: command
+short-summary: Triggers quick evaluation for acat report.
+examples:
+  - name: Simple example
+    text: |
+      az acat report download --report-name yourReportName --download-type "ComplianceReport"
+  - name: Download the file to specified path
+    text: |
+      az acat report download \\
+        --report-name yourReportName \\
+        --download-type "CompliancePdfReport"\\
+        --path "C:/workspace"\\
+        --name "out.pdf"
+"""
+helps['acat report get-control-assessments'] = """
+type: command
+short-summary: Triggers quick evaluation for acat report.
+examples:
+  - name: Simple example
+    text: |
+      az acat report get-control-assessments --report-name yourReportName
+  - name: Simple example
+    text: |
+      az acat report get-control-assessments --report-name yourReportName --compliance-status "failed"

```

Comment:
Can you have a try without double quota
```suggestion
      az acat report get-control-assessments --report-name yourReportName --compliance-status failed

```